### DISCUSSION TITLE
Randomly pick one from a list

### DISCUSSION THREAD

**Discussion by wdv4758h** [nushell/nushell#9147] [General]
Small sharing. I didn't find exactly command in `random ...` series that fit this directly. Not sure how do people do this, here is what I end up with:

```nu
def "random choose" [] {
    $in | get (random integer ..<($in | length))
}

[a b c d] | random choose
```

**Comment by amtoine** [comment]
i'd probably use `shuffle | first` :smirk:

**Reply by wdv4758h** [reply]
> i'd probably use `shuffle | first` :smirk: 

Great one!!!

**Reply by amtoine** [reply]
:blush:
